.changeset/tidy-radios-report.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": patch
+---
+
+CSS bug fix for hint renderer.
```

Comment:
**suggestion**: Could you be more explicit here about what is fixed?  Something like "Fix list item rendering in HintRenderer (CSS bug fix)" or something.  I'm thinking into the future when we or someone else reviews the CHANGELOG and reads this line.